src/components/pages/areas-manage/AreasManage.js

```diff
@@ -135,7 +135,7 @@ class AreasManage extends React.Component {
                     />
                 </div>
               </div>
-              <Button className="c-button">Save</Button>
+              <Button className="c-button" disabled={this.props.saving}>Save</Button>
```

Comment:
Actually we can save the area without a name now even if its required (at least it redirects to index) This could be a solution for this problem